### DISCUSSION TITLE
Email confirmation via `wp-activate.php`

### DISCUSSION THREAD

**Discussion by seeReadCode** [wpengine/faustjs#979] [Q&A]
My cms domain is redirecting `/wp-activate.php?key=ACTIVATION_KEY` requests to the headless domain which generates a 404.  This url comes in the link in the email confirmation sent to new users.

How do I fix this so that the WP/CMS domain activates the user / does not redirect to the Headless domain?  I've contacted WP Engine support but figured I'd ask here since I don't see relevant docs on WP Engine, Faust.js or issues here.

Thanks,
C

**Comment by josephfusco** [comment]
@seeReadCode You can exclude certain routes by using the solution [as seen here](https://github.com/wpengine/faustjs/pull/740#issuecomment-1090788893).

There is also a [new filter](https://github.com/wpengine/faustjs/pull/1320) coming in the next plugin update to make that a little cleaner!
